src/iproject.ts

```diff
@@ -1471,6 +1471,20 @@ export class IProject {
         }
       }
 
+      const buildMap = await this.getBuildMap();
+      if (buildMap) {
+        buildMap.forEach((ibmiJsonContent, path) => {
+          const objLib = ibmiJsonContent.build?.objlib;
+          if (objLib) {
+            const libraryType = objLibs.get(objLib);
+            objLibs.set(objLib, [
+              LibraryType.objectLibrary,
```

Comment:
Does this resolve variables?

Reply:
This function will return unresolved variables. They are resolved at the time of calling `getChildren` on the Object Libraries heading: https://github.com/IBM/vscode-ibmi-projectexplorer/blob/main/src/views/projectExplorer/objectlibraries.ts#L36